### PR TITLE
swagger2

### PR DESCRIPTION
**Pull request by GrzegorzSadowski**
- adding dependiences for swagger2 and swagger ui in pom.xml
- basic SwaggerConfig file

### CODE REVIEW

File: finance_manager/config/SwaggerConfig.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+                "0.1",
+                "Terms of service",
+                new Contact("John Doe", "exadel.com", "myeaddress@exadel.com"),
+                "License of API", "API license URL", Collections.emptyList());
+    }
+
+
+
+
+
```
Comment (serkushner) [inline finance_manager/config/SwaggerConfig.java:42]: Extra lines

File: finance_manager/pom.xml [hunk 2]
```diff
@@ -1,126 +1,139 @@
...
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.cloud</groupId>
+            <artifactId>spring-cloud-starter-contract-stub-runner</artifactId>
+            <version>3.1.3</version>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-data-jpa</artifactId>
+        </dependency>
```
Comment (serkushner) [inline finance_manager/pom.xml:75]: There is so much changed lines, pls make sure that you use the same code style within the team

File: finance_manager/config/SwaggerConfig.java (hunk 1 above)
Comment (GrzegorzSadowski) [reply finance_manager/config/SwaggerConfig.java]: Sure

File: finance_manager/pom.xml (hunk 2 above)
Comment (GrzegorzSadowski) [reply finance_manager/pom.xml]: Ok

**Review by serkushner — APPROVED** [review]

**Review by AlexPravda1 — APPROVED** [review]

**Pull request merged**